lib/Models/CatalogGroup.js

```diff
@@ -52,6 +52,15 @@ var CatalogGroup = function(terria) {
     this.isLoading = false;
 
     /**
+     * Gets or sets a value that tells the UI whether this is a group. 
+     * Groups, when clicked, expand to show their constituent items.
+     * This property is observable.
+     * @type {Boolean}
+     * @default false
+     */
+    this.isGroup = true;
```

Comment:
It would be better to define this property with the `defineProperties` section below, and make it gettable-only.  Defining it here means it is settable, and it will even be deserialized from JSON.